dev/ledger.md

```diff
@@ -796,6 +807,15 @@ An application call transaction additionally has the following fields:
 - Asset IDs that the executing program may read asset parameters from. This
   field is encoded as msgpack field `apas`. The maximum number of entries in
   this field is 8.
+- Box references that the executing program or any other program in
+  the same group may access for reading or modification when the
+  reference matches the running programs app ID. This field is encoded
+  as msgpack field `apbx`, each element of which is encoded as a
+  msgpack object containing an application ID (`i`) and name
```

Comment:
I may just be confused here but isn't the `i` actually the index into the `ForeignApps` array rather than the appID? I'm basing this off of the `transactions.BoxRef` object.

Reply:
You're right! I forgot my own encoding for a while.  I think it's correct elsewhere. But anyway, I'll clean it up, thanks.